app/assets/javascripts/study_level_activities.js.coffee

```diff
@@ -30,7 +30,7 @@ $ ->
       url: "/line_items/new"
       data: data
 
-  $(document).on 'change', '.components > .selectpicker', ->
+  $(document).on 'change', '.components .sla_components > .selectpicker', ->
```

Comment:
Small comment: Why use both `.components` and `.sla_components`? Seems `.sla_components` is plenty for the selector?

Reply:
It was mostly just to make the lowest impact change to fix the problem. The additional selector was needed, but leaving the higher level .components selector doesn't hurt anything. Less chance of causing unforeseen issues.

(I could go either way, I pretty much flipped a mental coin, since it doesn't matter really either way)